src/components/Carousel/Carousel.test.jsx

```diff
@@ -0,0 +1,8 @@
+import { customRender } from "../../utils/testUtils";
+import Carousel from "../Carousel/Carousel";
+
+it("should render the text on the display", () => {
+  const { container } = customRender(<Carousel />);
```

Comment:
Could you pass in some components to render here? Something really simple, to show this component in action